tests/testthat/test_recode2NA.R

```diff
@@ -27,13 +27,18 @@ test_that("Recode2NA mixed data and missings in string", {
 
 
 test_that("Errors for Recode2NA", {
-  expect_error(out <- recode2NA(txt_gads, value = c("", "la")), "'value' needs to be a vector of exactly length 1.")

```

Comment:
Nit: Maybe, an error-free test (with 2 or more `value`s) could also be added above?

Reply:
Agree and fixed!